client/src/components/Form/Form.js

```diff
@@ -8,12 +8,14 @@ const Form = ({ onAddQuestion }) => {
     setQuestion(event.target.value)
   }
 
-  const submitForm = () => {
+  const submitForm = event => {
+    event.preventDefault()
+    document.getElementById('create-question-form').reset()
```

Comment:
Please **don't** use native/vanilla JavaScript in React! You can get the element and set the focus/reset via the event through:
```
const form = event.target
const { newQuestion } = from.elements
form.reset()
newQuestion.focus()
```